### PR TITLE
Update for home Fragment and viewmodel

### PR DESCRIPTION
**Pull request by taehong3**
# Description
- Created basic homeFragment
- Put place some placeholders in homeFragment
- Add required strings in string.xml
- Created empty HomeViewModel.java 

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- Ran the application in simulator 

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -29,10 +29,10 @@ android {
 }
 
 dependencies {
-
     implementation 'androidx.appcompat:appcompat:1.2.0'
     implementation 'com.google.android.material:material:1.3.0'
     implementation 'androidx.constraintlayout:constraintlayout:2.0.4'
+    implementation 'androidx.legacy:legacy-support-v4:1.0.0'
```
Comment (gtullio12) [inline app/build.gradle:35]: What is this for?

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -6,4 +6,7 @@
     <string name="title_chat">Chat</string>
     <string name="title_weather">Weather</string>
 
+    <string name="hello_blank_fragment">Hello blank fragment</string>
```
Comment (gtullio12) [inline app/src/main/res/values/strings.xml:9]: Where is this string resource used?

File: app/src/main/java/ui/HomeViewModel.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+import androidx.annotation.NonNull;
+import androidx.lifecycle.AndroidViewModel;
+
+public class HomeViewModel extends AndroidViewModel {
+
+    public HomeViewModel(@NonNull Application application) {
+        super(application);
+    }
+}
```
Comment (gtullio12) [inline app/src/main/java/ui/HomeViewModel.java:13]: This is a very picky style suggestion but you should add a newline to the end of every file. https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file

File: app/src/main/java/ui/HomeFragment.java [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+        return inflater.inflate(R.layout.fragment_home, container, false);
+    }
+
+    @Override
+    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
+        super.onViewCreated(view, savedInstanceState);
+//        UserInfoViewModel model = new ViewModelProvider(getActivity())
+//                .get(UserInfoViewModel.class);
+    }
+}
```
Comment (gtullio12) [inline app/src/main/java/ui/HomeFragment.java:34]: This is a very picky style suggestion but you should add a newline to the end of every file. https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file

**Review by gtullio12 — COMMENTED** [review]
Looks good. No suggestions but just the minor comments I made. Also great pull request format!

**Comment by skim-93** [comment]
- I chekced it merged to the main-

**Pull request merged**